README.md

```diff
@@ -5,115 +5,72 @@
 
-### Run the tests
-
-#### Unit tests
-
-In a terminal `cd` into the directory you checked this project out into, then:
-
-```
-$ make test
-```
-
-To run a specific test, or set of tests, run:
+Once the API is running locally, run
 
 ```
-$ make test TEST=[testname]
+./run.sh start
 ```
 
-#### Integration/functional tests
+This will build everything (which will initially take a while) and run
+the local server at [http://localhost:8001](http://localhost:8001).
 
-Feature tests are run using a combination of [Cucumber.js](https://github.com/cucumber/cucumber-js) 
-and [WebdriverIO](http://webdriver.io/api.html). 
+### Alternative: Gulp
 
-Feature files are in `test/features/` and step definitions live in `test/features/steps/`.
-The WebdriverIO [testrunner](http://webdriver.io/guide/testrunner/gettingstarted.html) 
-is used to run the tests using a [standalone selenium server](https://www.npmjs.com/package/selenium-standalone).
+The method above is still experimental. The traditional way, which is closer
+to how real deployments are done, is:
 
-Config for the testrunner lives in `test/`.
-
-To run the tests against all currently selected browsers (Chrome, Firefox, PhantomJS), run:
 ```
-$ npm test
+virtualenv -p python3 venv
+source venv/bin/activate
+pip install -r requirements/dev.txt
+gulp
+./manage runserver 8001
 ```
 
-To run the tests headlessly, run:
-```
-$ npm run test-headless
-```
-
-You can also [tag](https://github.com/cucumber/cucumber/wiki/Tags) scenarios with `@wip` 
-and run the following command to only run those scenarios:
-```
-$ npm run test-wip
-```
+### Alternative: Docker
 
-### Validate code style
-
-#### Python
-
-In a terminal `cd` into the directory you checked this project out into, then:
+In order to run a server that's exactly similar to the production machines,
+you need to have [Docker](http://docs.docker.com/installation/mac/) and
+[Docker Compose](https://docs.docker.com/compose/install/) installed. Run
 
 ```
-$ make lint
+make run
 ```
 
-To check for a [specific class of style
-violation](http://flake8.readthedocs.org/en/latest/warnings.html), run:
+and you should eventually be able to connect to the local server.
 
-```
-$ make lint LINT_OPTS="--select [lint-rules]"
-```
+### Log in to the application
 
-#### JavaScript
+You should be able to log into the cash book app using following credentials:
 
-To lint javascript files using [JSHint](http://jshint.com/), run:
-```
-$ gulp lint
-```
+- *test-prison-1 / test-prison-1* for Prison 1
+- *test-prison-2 / test-prison-2* for Prison 2
 
-### Run a development Server
+## Developing
 
-In a terminal `cd` into the directory you checked this project out into, then
+With the `run.sh` command, you can run a browser-sync server, and get the assets
+to automatically recompile when changes are made, run `./run.sh serve` instead of
+`./run.sh start`. The server is then available at [http://localhost:3000](http://localhost:3000)
```

Comment:
actually, the port will vary per-application so maybe you can say that it will print the url that it's serving on